CrossRefExportPlugin.php

```diff
@@ -388,6 +388,10 @@ public function updateDepositStatus($context, $object, $status, $batchId = null,
                 $this->getSuccessMsgSettingName() => $successMsg,
             ];
 
+            if ($status === Doi::STATUS_REGISTERED) {
+                $editParams['registrationType'] = $this->getName();
+            }
```

Comment:
Thanks, @NateWr. For choosing which agencies should be among those "known agencies" passed to `DoiListPanel`, what do you think of only including agencies with plugins currently installed and enabled (e.g. via a hook similar to how the selected registration agency is handled in the DOI settings)? 

This would simplify knowing which agencies exist, but would potentially leave unlocalized strings if the plugin is ever disabled. Or would it be better to hard-code in a few of the most used registration agencies (since they have been fairly stable over time)?

Reply:
Yes, that was my thinking: all enabled registration agencies in the context, with a fallback where the agency isn't recognized (you can use the machine-readable agency id, which will likely give some information).